documentation/release_notes.markdown

```diff
@@ -109,131 +162,150 @@ stable release of PuppetDB, which included an experimental v4 API.
 
-* Retire `[database]` configuration key node-ttl-days ([PDB-644](https://tickets.puppetlabs.com/browse/PDB-644))
+* Retire `[database]` configuration key `node-ttl-days` ([PDB-644](https://tickets.puppetlabs.com/browse/PDB-644))
 
-* Retire `[jetty]` configuration key certificate-whitelist ([PDB-886](https://tickets.puppetlabs.com/browse/PDB-886))
+* Retire `[jetty]` configuration key `certificate-whitelist` ([PDB-886](https://tickets.puppetlabs.com/browse/PDB-886))
 
     This now lives in the `[puppetdb]` section.
 
-* PuppetDB now uses the Ezbake build system ([PDB-663](https://tickets.puppetlabs.com/browse/PDB-663))
-
-    * The old system is still around, until we work out how to do source
-      based installations of PuppetDB in this new world. This will be done as
-      another piece of work separately.
-    * Use the nrepl trapperkeeper service. We provide a warning if
-      users use the old `[repl]` block. Instead users must define the
-      `[nrepl]` block, and it is no longer possible to choose a `type`
-      of repl service.
-    * Binaries are now installed in /usr/bin instead of /usr/sbin as per the ezbake
-      standards.
-    * Lots of template files were transferred into resource/ext, this is what ezbake
-      expects.
-    * Documentation has moved inside /usr/share/puppetdb/docs to a new location
-      depending on the distribution. This is to follow the ezbake standards.
-    * Many symlinks were removed from /usr/share/puppetdb that were pointing
-      at the /var/lib/puppetdb and /etc/puppetdb/ spaces. This finishes a partially
-      completed migration from the past.
+* PuppetDB now has a new filesystem layout that complies with the overall Puppet AIO layout ([PDB-1455](https://tickets.puppetlabs.com/browse/PDB-1455))
 
-### New Features
+* PuppetDB now uses the Ezbake build system which changes the way we package ([PDB-663](https://tickets.puppetlabs.com/browse/PDB-663))
 
-* Add support for hyphenated classnames, supported by Puppet 2.7.x - 3.7.x. ([PDB-1024](https://tickets.puppetlabs.com/browse/PDB-1024))
+* Due to the Ezbake changes, the `[repl]` block has been renamed to `[nrepl]` and `type` is no longer an accpted parameter ([PDB-663](https://tickets.puppetlabs.com/browse/PDB-663))
 
-* Add a '-v' flag to the CLI to get the PuppetDB version.
+* The endpoints contained in `/experimental` have now been removed ([PDB-1127](https://tickets.puppetlabs.com/browse/PDB-1127))
 
-* The `reports` database table was using the hash string as its
-  primary key, we know have switched to using a smaller bigint primary
-  key for that table, which should result in faster joins in most
-  cases, and hopefully smaller foreign key indexes sizes for tables
-  relating to
-  `reports`. ([PDB-1218](https://tickets.puppetlabs.com/browse/PDB-1218))
+* Return proper `404` errors for child endpoints where there parents do not exist, instead of empty lists ([PDB-1473](https://tickets.puppetlabs.com/browse/PDB-1473))
```

Comment:
we should define "child endpoints" somewhere. In previous docs I think we've informally called them "shorthand endpoints".